iceoryx_hoofs/test/moduletests/test_cxx_function.cpp

```diff
@@ -53,13 +53,13 @@ class Counter
         ++numCreated;
     }
 
-    Counter(const Counter&)
+    Counter(const Counter& rhs IOX_MAYBE_UNUSED)
```

Comment:
Nitpick: Could also be
```suggestion
    Counter(const Counter&)
```

Reply:
No then clang-tidy prints out a warning!

Reply:
Yes but I think this is due to `hicpp-named-parameter` which I am in favor of disabling (have it disabled in https://github.com/eclipse-iceoryx/iceoryx/pull/1475/files#diff-0534891fbc8b89b4c521057e7e8dff70d3cbeb822cc9991a3abad77848c289f8). It does more harm than good and is too restrictive. The compiler forces us to have names where they are required anyway; it is impossible to circumvent it for variables that are really used, and for others it should not be mandatory (due to cases like this).